src/components/TagEditor.tsx

```diff
@@ -70,10 +80,16 @@ export function TagEditor(props: {
             />
             <div class="flex flex-wrap gap-2">
```

Comment:
what if we just drop this whole section? it's supposed to update with recommended tags as you use them, but right now it's just kind of confusing

Reply:
Went ahead and completely removed it. maybe we should poll if we want it as is it remove it until the redesign